### DISCUSSION TITLE
How can I "inline" html inputs ?

### DISCUSSION THREAD

**Discussion by andreElrico** [codex-team/editor.js#1923] [Q&A]
Is it possible to create something like this with editorjs?

my ideas:

+ make an line tool that renders inputs

any ideas or tricky things I should consider ?

==> https://github.com/bitstat/paperstencil

![image](https://camo.githubusercontent.com/339a6114d54a566940775890b58acd32065b691cd2acdddd5cbcd4969194031c/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f70617065727374656e63696c2f6173736574732f776f726470726f636573736f722e676966)

**Comment by hata6502** [comment]
My experimental plugin may help you: 
https://github.com/hata6502/editorjs-inline
